src/main/java/mSearch/daten/ListeFilme.java

```diff
@@ -50,18 +51,18 @@
     public int nr = 1;
     public String[] metaDaten = new String[]{"", "", "", "", ""};
     private final static String DATUM_ZEIT_FORMAT = "dd.MM.yyyy, HH:mm";
-    private final static String DATUM_ZEIT_FORMAT_REV = "yyyy.MM.dd__HH:mm";
-    SimpleDateFormat sdf = new SimpleDateFormat(DATUM_ZEIT_FORMAT);
+    //private final static String DATUM_ZEIT_FORMAT_REV = "yyyy.MM.dd__HH:mm";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)